umbral/point.py

```diff
@@ -264,6 +266,9 @@ def unsafe_hash_to_point(data, params, label=None):
             else:
                 # Any other exception, we raise it
                 raise e
+        except ValueError as e:
```

Comment:
We should catch the `InternalErrors` above within `Point.from_bytes` and raise a unified `ValueError` instead, and catch that here.